doc/sources/guide/lang.rst

```diff
@@ -55,54 +55,55 @@ App instance::
     Widget:
 
 A `class` rule, declared by the name of a widget class between `< >` and
-followed by `:`, defines how any instance of that class will be
-graphically represented::
+followed by `:`, defines the appearance and behavior of any instance of that
+class::
 
     <MyWidget>:
 
-Rules use indentation for delimitation, as python, indentation should be of
-four spaces per level, like the python good practice recommendations.
+Rules use indentation for delimitation, like Python. Indentation should be
+four spaces per level, like the Python style guide
+`recommends <https://www.python.org/dev/peps/pep-0008/#indentation>`_.
 
-There are three keywords specific to Kv language:
+There are three keywords specific to the Kv language:
 
 - `app`: always refers to the instance of your application.
 - `root`: refers to the base widget/template in the current rule
 - `self`: always refer to the current widget
 
-Special syntaxes
+Special syntax
 ----------------
 
-There are two special syntaxes to define values for the whole Kv context:
+There is a special syntax to define values for the whole Kv context.
 
-To access python modules and classes from kv, ::
+To access Python modules and classes from kv, use ``#:import`` ::
 
     #:import name x.y.z
     #:import isdir os.path.isdir
     #:import np numpy
 
-is equivalent to::
+is equivalent to
 
     from x.y import z as name
     from os.path import isdir
     import numpy as np
 
-in python.
+in Python.
 
-To set a global value, ::
+To set a global value, use ``#:set`` ::
 
     #:set name value
 
-is equivalent to::
+is equivalent to
 
     name = value
 
-in python.
+in Python.
 
 Instantiate children
 --------------------
 
-To declare the widget has a child widget, instance of some class, just declare
-this child inside the rule:
+To declare the widget has a child widget that is an instance of some specific
```

Comment:
`declare the widget has a child widget` should be `declare the widget as a child widget` right?
So I mean: `s/has/as/`